tests/Fixtures/ConvertConfig/class_references.yaml

```diff
@@ -0,0 +1,25 @@
+default:
+    extensions:
+        # Loads the custom classes we need for this feature
+        class_references_loader.php: ~
+        # Is referenced exactly by its class name, and the class is defined, so will be imported & referenced
+        Some\Behat\Extension\ExplicitlyReferencedExtension: ~
+        # Uses the "automagic" Some\ShorthandExtension => Some\ShorthandExtension\ServiceContainer\ShorthandExtension
+        # Is not converted to a class reference
```

Comment:
This is actually converted to a class reference so this comment does not seem correct

Reply:
Ah yes, correct - I had added like that on the first "convert class refs but not shorthand" commit and forgot to change it in the second "also convert shorthand" commit. Fixed now.